sortingview/experimental/SpikeSortingView/MultiTimeseries.py

```diff
@@ -0,0 +1,40 @@
+import os
+from typing import List, Union
+import kachery_client as kc
+from .Figure import Figure
+
+
+class MultiTimeseries:
+    def __init__(self, *, label: str) -> None:
+        self._label = label
+        self._panels = []
+    def add_panel(self, figure: Figure, *, relative_height: Union[float, None]=None):
+        self._panels.append({
+            'figure': figure,
+            'relative_height': relative_height
+        })
```

Comment:
So in practice, `add_panel` is used to add a timeseries (figure) along with its relative height to the list of panels that should be presented together. Then, when we want to render them, we call `get_composite_figure` which collects the individual figures and labels the collection as a `MultiTimeseries`, which is what gets displayed on the front-end. Right?

Reply:
That's right.